src/main/java/com/couchbase/intellij/tree/cblite/CBLTreeRightClickListener.java

```diff
@@ -89,58 +100,114 @@ public void actionPerformed(@NotNull AnActionEvent e) {
         showPopup(e, tree, actionGroup);
     }
 
-    private static void handleDocument(Project project, MouseEvent e, DefaultMutableTreeNode clickedNode, CBLFileNodeDescriptor userObject, Tree tree) {
+    private static void handleDatabase(Project project, MouseEvent e, DefaultMutableTreeNode clickedNode,
+            CBLDatabaseNodeDescriptor userObject, Tree tree) {
         DefaultActionGroup actionGroup = new DefaultActionGroup();
-        AnAction deleteDocument = new AnAction("Delete Document") {
+
+        boolean isActive = userObject.getDatabase().getId().equals(ActiveCBLDatabase.getInstance().getDatabaseId());
+        if (isActive) {
+            AnAction menuItem = new AnAction("Disconnect") {
+                @Override
+                public void actionPerformed(@NotNull AnActionEvent e) {
+
+                    try {
+                        CBLTreeHandler.disconnectFromCluster(clickedNode, userObject, tree);
+                    } catch (CouchbaseLiteException ex) {
+                        Log.error(ex);
+                        SwingUtilities
+                                .invokeLater(() -> Messages.showErrorDialog("Could not disconnect from the database.",
+                                        "Couchbase Lite Connection Error"));
+                    }
+                }
+            };
+            actionGroup.add(menuItem);
+
+            AnAction createCollection = new AnAction("Create Scoped Collection") {
```

Comment:
Avoid creating new terms, "Scoped Collection" is not something that mentioned anywhere in our documentation. Rename to "Create Collection"